sessions/session-b.md

```diff
@@ -0,0 +1,36 @@
+# Session B: Creating low-power sensor networks using LoRa
+
+## Description
+
+The last few year have been an exciting time for the DIY electronics community. There has been an explosion of sensors, actuators, and micro-controllers to tinker with. Additionally, with access to inexpensive hardware like the Photon, Raspberry Pi, and ESP8266 it’s easier than ever to embed web-services directly in to our projects and this connectivty enables new applications. However, most of these projects connect over WiFi, which isn't well suited for low-power/low-bandwidth applications. WiFi usually limits our projects to indoors or tethered to our smartphones. These mobility and power limitations impede projects like distributed citizen science, connected guerrilla art, and remote sensing. LoRa is an inexpensive low-power/low-bandwidth radio technology that is one great solution to this problem. In urban settings LoRa radios can often provide low bandwidth connectivity of up to 1-2 kms and can be reasonably powered with AA batteries.
```

Comment:
- last few _years_
- _ESP8266,_
- directly _into_ our projects
- In urban _settings,_
- 1-2 _km_